src/index.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from "./node";
 import { RDTRootNode, ROOT_NODE_TAG_MAP, fetchRootNode } from "./root";
 
-import { nanoid } from "nanoid";
+import { nanoid } from "nanoid/non-secure";
```

Comment:
Would rather not use unsafe IDs, trackweave is heavily dependent on the idea that it is highly unlikely for a repeat of an id. Making it unsafe could violate this.